packages/vscode-extension/src/officeChat/commands/create/helper.ts

```diff
@@ -95,11 +108,11 @@ export function getOfficeTemplateMetadata(): ProjectMetadata[] {
 export async function showOfficeSampleFileTree(
   projectMetadata: ProjectMetadata,
   response: ChatResponseStream
-): Promise<string> {
+): Promise<ProjectMiniData> {
   response.markdown(
     "\nWe've found a sample project that matches your description. Take a look at it below."
   );
-  const downloadUrlInfo = await getOfficeSampleDownloadUrlInfo(projectMetadata.id);
+  const { downloadUrlInfo, host } = await getOfficeSampleDownloadUrlInfo(projectMetadata.id);
```

Comment:
How about the host type of the matched templates? seems like there's only telemetry for matched samples.

Reply:
it has been already logged. thanks